docs/apps/games/minecraft-java/Comunity_Guide.md

```diff
@@ -0,0 +1,51 @@
+# Minecraft Java Community Guide
+
+\*Warning This guide contains information that uses Advanced/Expert settings. As a result this will be outside the scope of support!
```

Comment:
```suggestion
⚠️ **Warning This guide contains information that uses Advanced/Expert settings. As a result this will be outside the scope of support!** ⚠️
```

Reply:
We can make it clear as day LoL